modules/cnf-about-ptp-fast-event-notifications-framework.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-ptp.adoc
+
+[id="cnf-about-ptp-fast-event-notifications-framework_{context}"]
+= About the PTP fast event notifications framework
+
+DU applications can subscribe to PTP fast events notifications that are generated by {product-title} with the PTP Operator and `cloud-event-proxy` sidecar container deployed. An AMQP event bus is also required.
```

Comment:
"An AMQP event bus is also required"
"AMQ Interconnect  operator need to be installed"
AMQ Interconnect is based on Dispatch Router from the Apache Qpid™ project.